### PR TITLE
Soback 70 show pwid

### PR DESCRIPTION
**Pull request by jorntx**
Small component to show the PWID xml. For now the styling is crude as it is the exact format (as displayed) that is interesting to the end-user. Maybe something can be done about that later.

Egense added as co-reviewer to ensure that the component delivers the format correctly and it is as needed by the researchers

### CODE REVIEW

File: src/js/src/router/index.js [hunk 1]
```diff
@@ -44,6 +44,15 @@ const routes = [
...
       import(/* webpackChunkName: "page-harvest-data" */ '../views/PageHarvestData.vue')
+  },
+  {
+    path: '/pwid',
+    name: 'PWID',
+    meta: {
+      title: 'PWID'
+    },
+    component: () =>
+      import(/* webpackChunkName: "page-harvest-data" */ '../views/PWID.vue')
```
Comment (jesperlauridsen) [inline src/js/src/router/index.js:55]: probably the wrong comment? Its no longer "page-harvest-data" ;)

**Review by thomasegense — APPROVED** [review]
Approved with the new formatting change.

File: src/js/src/router/index.js (hunk 1 above)
Comment (jorntx) [reply src/js/src/router/index.js:55]: Damn you copy/paste - thanx:)

**Review by jesperlauridsen — APPROVED** [review]
I just spun it up - looks great to me! Good work

**Pull request merged**